apps/crawler/src/skd/sdk.service.ts

```diff
@@ -8,14 +8,15 @@ import {
   TokenByIdResult,
   TokenPropertiesResult,
 } from '@unique-nft/sdk/tokens';
+import { ConfigService } from '@nestjs/config';
```

Comment:
'skd' in file path looks like typo.

Reply:
<img width="335" alt="image" src="https://user-images.githubusercontent.com/22288830/184842447-f92564fb-da8f-4972-b011-531d10560485.png">

Reply:
Yopt!

Reply:
fixed